typings/custom-window-defs.d.ts

```diff
@@ -11,4 +11,10 @@ interface Window {
     Base64Binary: Base64Binary;
     jQuery: JQueryStatic;
     dataLayer: object[];
+    // eslint-disable-next-line oppia/disallow-flags
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    GifFrames: any;
+    // eslint-disable-next-line oppia/disallow-flags
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Guppy: any;
```

Comment:
Please add explanations for these ignores

Reply:
I guess it was a last ditch attempt to fix types. I will remove this.

Reply:
Done